### PR TITLE
Merging Develop into Stable

### PR DESCRIPTION
**Pull request by drnimbusrain**
There are more changes upcoming, and like to get these recent develop changes into stable.

### CODE REVIEW

File: python/README.md [hunk 1]
```diff
@@ -51,3 +51,35 @@ cases = config_cases(
...
+   frp_src  = 1      # frp data source (0: local source; 1: check local source first, switch to climatological file if no available data; 2: 12 month climatology; 3: all ones when ifcanwaf=.FALSE.)
+   ```
+
+2. Activate canopy-app Conda environment:
+
+   ```
+   conda activate canopy-app
+   ```
+
+3. Run python script
```
Comment (zmoon) [inline python/README.md]:
```suggestion
3. Run Python script
```

File: src/Makefile [hunk 2]
```diff
@@ -14,12 +14,16 @@ $(info FC setting: '$(FC)')
...
 
 # Default to NetCDF build
-NETCDF ?= 1
+NC ?= 1
 
 # Compile flags
 $(info DEBUG setting: '$(DEBUG)')
 ifeq ($(DEBUG), 1)
-  FCFLAGS := -g -Wall -Wextra -Wconversion -Og -pedantic -fcheck=bounds -fmax-errors=5
+  ifeq ($(FC),gfortran)
```
Comment (zmoon) [inline src/Makefile]: I noticed issue with this: your FC must be exactly `ifort` or `gfortran` or you won't get debug flags currently. For example `gfortran-12`, `/usr/bin/gfortran`, won't be recognized. CMake will help make solving this easier and more robust. But for this PR we might want to fix it for the CI case (`gfortran-12`-like), maybe using `notdir` and `findstring`.

Comment (zmoon) [reply src/Makefile]:
```diff
 # Compile flags
 $(info DEBUG setting: '$(DEBUG)')
 ifeq ($(DEBUG), 1)
-  ifeq ($(FC),gfortran)
+  ifeq ($(findstring gfortran,$(notdir $(FC))),gfortran)
     FCFLAGS := -g -Wall -Wextra -Wconversion -Og -pedantic -fcheck=bounds -fmax-errors=5 -std=f2003 -fall-intrinsics
   else ifeq ($(FC),ifort)
     FCFLAGS := -g -warn all -check bounds -implicitnone -O0 -error-limit 5
@@ -29,6 +29,7 @@ else ifeq ($(DEBUG), 0)
 else
   $(error invalid setting for DEBUG, should be 0 or 1 but is '$(DEBUG)')
 endif
+$(info FCFLAGS: '$(FCFLAGS)')

 # NETCDF Settings here
 LIBS :=
```

Comment (drnimbusrain) [reply src/Makefile]: Thank you Zach!  Made the [change](https://github.com/noaa-oar-arl/canopy-app/pull/84/commits/d7cfa3a139f06728748e1c0d0dac9e651cde5835), please let me know!  
```

Comment (zmoon) [reply src/Makefile]: Looks like it is working now!

File: README.md [hunk 3]
```diff
@@ -19,12 +19,23 @@ Canopy-App requires NetCDF-Fortran Libraries (i.e., `-lnetcdf -lnetcdff`) when u
 See [the included Makefile](./src/Makefile), which detects NetCDF using `nf-config`, for an example (on GMU Hopper, you can use the `netcdf-c/4.7.4-vh` and `netcdf-fortran/4.5.3-ff` modules).
 
 Compilation options can be controlled with environment variables:
- - `DEBUG=0` (off; default) or `DEBUG=1` (on)
- - `NETCDF=0` (off) or `NETCDF=1` (on; default)
+
+- `FC=gfortran` (default) or ifort
```
Comment (zmoon) [inline README.md]:
```suggestion
- `FC=gfortran` (default) or compiler name/path (e.g. `FC=ifort`, `FC=gfortran-11`, `FC=/usr/bin/gfortran-11`)
```

**Review by zmoon — APPROVED** [review]
Let's do it

**Pull request merged**